CMake 2.8.12 support.

It does not make sense to add MYFLTK_LIBRARIES (which translates to
something like "-L/usr/local/lib -lfoo -lbar") to the link interface
library of a static library like libzynaddsubfx_gui.a.

Removing FLTK_LIBRARIES does not cause any problems either, as
GUI_LIBRARIES in src/CMakeLists.txt sets all the required GUI-related
libraries that need to be linked against the zynaddsubfx binary.
--- src/UI/CMakeLists.txt.orig	2012-06-10 22:20:18 UTC
+++ src/UI/CMakeLists.txt
@@ -32,5 +32,3 @@ add_library(zynaddsubfx_gui STATIC
         NioUI.cpp
         WidgetPDial.cpp
 	)
-
-target_link_libraries(zynaddsubfx_gui ${FLTK_LIBRARIES} ${MYFLTK_LIBRARIES})
